test/end-to-end/offer.js

```diff
@@ -516,7 +517,7 @@ describe("Offer endpoint tests", () => {
 
             beforeAll(async () => {
 
-                await Company.deleteMany({});
+                // await Company.deleteMany({});
```

Comment:
Why are you not deleting?

Reply:
I wanted first to hear feedback

Reply:
I meant deleting as calling the `deleteMany()`, why did you remove it? Every describe should delete + recreate the required data